### PR TITLE
Feature/EMPUII-3192 Drang-n-frop feature

### PR DESCRIPTION
**Pull request by AleksandrUSA96**

### CODE REVIEW

File: src/ui/UiApp.module.css [hunk 1]
```diff
@@ -0,0 +1,78 @@
...
+.app {
+  position: relative;
+}
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
+  align-items: center;
+  position: relative;
```
Comment (jellySWATy) [inline src/ui/UiApp.module.css:15]: I think unnecessary here, correct me if i'm wrong

File: src/ui/UiApp.module.css [hunk 2]
```diff
@@ -0,0 +1,78 @@
...
+  top: 25px;
+  left: 25px;
+  z-index: 1000;
+  margin-bottom: 25px;
+  width: calc(100% - 50px);
+}
+
+.top {
+  display: flex;
+  flex-direction: row;
```
Comment (jellySWATy) [inline src/ui/UiApp.module.css:25]: It's default flex value, unnecessary here

File: src/ui/UiMain2d.js [hunk 3]
```diff
@@ -72,6 +72,8 @@ class UiMain2d extends React.Component {
       store.dispatch({ type: StoreActionType.SET_SLIDER_2D, sliceRatio: valNormalizedTo01 });
       // clear all 2d tools
       const gra2d = store.graphics2d;
+      console.log(store);
+      console.log(gra2d);
```
Comment (jellySWATy) [inline src/ui/UiMain2d.js:76]: Is there debug console log? Fix if you can

**Review by jellySWATy — APPROVED** [review]

File: src/ui/UiApp.module.css [hunk 4]
```diff
@@ -0,0 +1,78 @@
...
+ */
+
+.app {
+  position: relative;
+}
+
+.header {
+  display: flex;
+  flex-wrap: nowrap;
+  flex-direction: row;
```
[the remark this answers is not in the document]
Comment (AleksandrUSA96) [reply src/ui/UiApp.module.css:13]: flex-wrap: nowrap; flex-direction: row; - unnecessary here, but flex need here.

[the remark this answers is not in the document]
Comment (AleksandrUSA96) [reply src/ui/UiApp.module.css:13]: > It's default flex value, unnecessary here

flex-wrap: nowrap; flex-direction: row; - unnecessary here, but flex need here.

**Pull request merged**